common/oauthTokenManager.go

```diff
@@ -80,8 +88,8 @@ func newAzcopyHTTPClient() *http.Client {
 			Proxy: GlobalProxyLookup,
 			// We use Dial instead of DialContext as DialContext has been reported to cause slower performance.
 			Dial /*Context*/ : (&net.Dialer{
-				Timeout:   30 * time.Second,
-				KeepAlive: 30 * time.Second,
+				Timeout:   10 * time.Second,
```

Comment:
QQ: Why are we reducing the Timeout and KeepAlive time? Can you add a comment to explain it?

Reply:
We need to set the Timeout attribute to some small value in case there's some process listening on 40342 (arc imds port) which does cause the TCP connection to complete (no ECNONNREFUSED error) but it doesn't respond to the HTTP GET request. With the current client, we'll wait for ever.
Hence, it was proposed that we use a small timeout, 5 secs, since we are talking to local processes.